### PR TITLE
[RM Ch10.3.4] Updates

### PR DESCRIPTION
**Pull request by sureshkrishnan**
Expanding the text regarding the need for control user plane separation for efficient use of acceleration hardware. Fix for #1861

### CODE REVIEW

**Review by walterkozlowski — APPROVED** [review]

**Review by TFredberg — APPROVED** [review]
Relevant addition

File: doc/ref_model/chapters/chapter10.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ Over the past few years there has been a significant move towards decomposing ne
...
+While generic server hardware capabilities can be exclusively used for handling networking related workloads, this strategy is neither performant nor energy efficient. There are several forms of accelerators such as smart NICs, programmable networking fabrics/switches, and GPUs that can offload some of these workloads in order to provide higher throughput, energy efficiency and lower latency. The acceleration hardware is typically optimized for specific kinds of workloads and may not be able to host all kinds of workloads that can be hosted on generic server hardware (such is the case for several control plane workloads). Hence some form of disaggregation might be required to separate control and user plane responsibilities between generic server hardware and accelerated hardware in order to make efficient use of these accelerators. There is also a need for workload orchestration to be able to understand the packet acceleration needs of specific workloads and schedule such workloads on infrastructure with the requisite capabilities.
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter10.md:75]: The disaggregation is not only between control and user plane but also withing user plane workloads.

Would the generic and accelerated hardware needs to be distinguished by some means or through discovery?

**Review by pgoyal01 — APPROVED** [review]
Approve

**Comment by scottsteinbrueck** [comment]
PR #1985 is already merged, and the content is a superset of PR #1982.  

PR #1982 is being closed without merge.

**Pull request closed without merging**